voluncheer/voluncheer/settings.py

```diff
@@ -55,6 +58,7 @@
     "crispy_bootstrap5",
     "storages",
     "channels",
+    "supervisor",  # Not sure if supervisor is needed here.
```

Comment:
If you aren't sure we should probably remove it and only add it once we are convinced we need it. It's easier to add code than remove it.

Reply:
I have the config file for supervisor anyway. I think we should leave it there?

Reply:
But what is it doing? Why is it there?